scrapers/covid-19/govScrapers/getMexico.js

```diff
@@ -43,30 +43,48 @@ const formData = {
 	nationalCases: querystring.stringify({ cve: '000', nom: 'nacional', sPatType: 'Confirmados' }),
 	nationalDeaths: querystring.stringify({ cve: '000', nom: 'nacional', sPatType: 'Defunciones' }),
 	stateCases: querystring.stringify({ cve: '', nom: '', sPatType: 'Confirmados' }),
-	stateDeaths: querystring.stringify({ cve: '', nom: '', sPatType: 'Defunciones' })
+	stateDeaths: querystring.stringify({ cve: '', nom: '', sPatType: 'Defunciones' }),
+	stateSuspects: querystring.stringify({ cve: '', nom: '', sPatType: 'Sospechosos' }),
+	stateNegatives: querystring.stringify({ cve: '', nom: '', sPatType: 'Negativos' }),
+	nationalActNegSusRec: querystring.stringify({ cve: '000', nom: '', sPatType: '' })
 };
 
+/**
+ * Filter items by either today's date or yesterday's
+ * @param {Object} data	An object with a "date" property
+ * @returns {Array}	Array filtered by date
+ */
+// eslint-disable-next-line consistent-return
 const filterByDate = (data) => {
 	const date = new Date();
-	date.setDate(date.getDate() - 1);
-	return data.filter(item => item.date === `${date.getFullYear()}-${(date.getMonth() + 1).toString().padStart(2, '0')}-${date.getDate().toString().padStart(2, '0')}`);
+	const filtered = () => data.filter(item => item.date === `${date.getFullYear()}-${(date.getMonth() + 1).toString().padStart(2, '0')}-${date.getDate().toString().padStart(2, '0')}`);
+	const decrementDate = () => date.setDate(date.getDate() - 1);
+	try {
+		return filtered().length ? filtered() : decrementDate(date) && filtered();
+	} catch (err) {
+		logger.err(err);
+	}
```

Comment:
I think it would be better to return as well as log an error. This way you can handle errors down the chain of function calls better - perhaps return `null`